src/backend/storeManagers/hyperplay/games.ts

```diff
@@ -495,7 +539,7 @@ export async function install(
 
     const installVersion = releaseVersion ?? gameInfoVersion ?? '0'
 
-    if (platformToInstall === 'Browser') {
+    if (platformToInstall === 'Browser' && !wasPaused) {
```

Comment:
I can't think of a scenario where we would be resuming an installation of a browser game since browser games aren't downloaded and install basically just sets the install url. This also goes back to my previous point about simplifying the code here

Reply:
Generally this should be called if after paused, that is why allowing only few places to let this go further, and calls the extraction later as intended

Reply:
Done